apps/box/src/components/Cards/CardCarousel.tsx

```diff
@@ -32,6 +32,9 @@ export const CardCarousel = ({
         loop={false}
         width={(WINDOW_WIDTH - CARD_SPACING) * SCALE}
         height={height}
+        panGestureHandlerProps={{
+          activeOffsetX: [-10, 10],
+        }}
```

Comment:
I don't understand why the X axis relates to vertical scrolling. But checked the docs and this is the way it's done.

Reply:
Oh, I see so these are limit in the x-direction which are then passed down to the Carousel once exceeded. I thought it was the other way round that when these limits are exceeded they are passed to the `scrollView`. Must have miss-understood the docs